.eslintrc.js

```diff
@@ -27,16 +26,17 @@ module.exports = {
   rules: {
     'no-console': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
     'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
-    'max-len': ['warn', { code: 120 }],
+    'max-len': ['warn', { code: 120, ignoreRegExpLiterals: true }],
     'vue/attribute-hyphenation': 'off',
-    'vue/multi-word-component-names': 'off',
+    'vue/no-deprecated-filter': 'warn',
+    'vue/no-deprecated-slot-scope-attribute': 'warn',
+    'vue/no-deprecated-v-on-native-modifier': 'warn',
     'vue/no-v-for-template-key-on-child': 'warn',
-    'vue/no-v-model-argument': 'warn',
+    'vue/no-v-html': 'off',
```

Comment:
I recommend against turning this off, which disables this check entirely.

I think it's better to see a warning and have to consciously ignore it.

If there is usage of v-html that is perfectly fine then I recommend disabling it in that code, but just not globally.